pkg/conversion/generator.go

```diff
@@ -700,6 +701,37 @@ func (g *generator) existsConvertionFunction(inType, outType reflect.Type) bool
 	return false
 }
 
+func (g *generator) existsDedicatedConversionFunction(inType, outType reflect.Type) bool {
+	if inType == outType {
+		// Assume that conversion are not defined for "deep copies".
+		return false
+	}
+
+	if g.existsConversionFunction(inType, outType) {
+		return true
+	}
+
+	// TODO(wojtek-t): We should somehow change the conversion methods registered under:
+	// pkg/runtime/scheme.go to implement the naming convention for conversion functions
+	// and get rid of this hack.
+	type typePair struct {
+		inType  string
+		outType string
+	}
+	defaultConversions := []typePair{
+		{"[]runtime.RawExtension", "[]runtime.Object"},
+		{"[]runtime.Object", "[]runtime.RawExtension"},
+		{"runtime.RawExtension", "runtime.EmbeddedObject"},
+		{"runtime.EmbeddedObject", "runtime.RawExtension"},
+	}
+	for _, conv := range defaultConversions {
+		if conv.inType == g.typeName(inType) && conv.outType == g.typeName(outType) {
```

Comment:
That's what I wanted to do, but importing "pkg/runtime" here creates an "import cycle" and thus I couldn't do it. If you know how do workaround it, I happy to change it.